modules/rosapps/applications/fraginator/DriveVolume.cpp

```diff
@@ -1,11 +1,11 @@
 #include "DriveVolume.h"
 
 
-DriveVolume::DriveVolume ()
+DriveVolume::DriveVolume()
 {
     Handle = INVALID_HANDLE_VALUE;
     BitmapDetail = NULL;
-    return;
+    ZeroMemory(&Geometry, sizeof(Geometry));
```

Comment:
So this is the actual fix, the rest is just changing code to your preferred style?

Reply:
No. There are several instances where structs needed to be initialized with ZERO. Instead of writing malloc() and zeroMemory() in all these cases I just used HeapAlloc with ZeroMemory flag. :)
If there is another way, we could use the alternative. AFAIK a flag in VS/MSBuild exists to zero-initialize vars by default. If we could use a similar flag, then we would solve the problems without changing the code at all. :)